.eslintrc.yaml

```diff
@@ -1,32 +1,31 @@
 # ---
     parser: babel-eslint
-    # extends: "@cerner/eslint-config-terra"
-    settings: 
-        polyfills: 
+    extends: "@cerner/eslint-config-terra"
+    settings:
+        polyfills:
             - Promise
             - Map
-
-    rules: 
-        react/forbid-prop-types: off  
+    env:
+      jasmine: true
+    rules:
+        react/forbid-prop-types: off
 
     overrides:
         - files: "*.js"
-          rules: 
-            import/no-cycle: off
-            no-param-reassign: off
-            template-curly-spacing: off
-            indent: 
-                - error
-                - 2
-                - ignoredNodes:
-                    - TemplateLiteral
-            strict: off
-            func-names: off
-            no-undef: off
-            no-unused-expressions: off
-            no-new: off
-            no-underscore-dangle: off
-            no-console: off
+          rules:
+           import/no-cycle: off
+           template-curly-spacing: off
+           indent:
+               - error
+               - 2
+               - ignoredNodes:
+                  - TemplateLiteral
+           strict: off
+           func-names: off
+           no-underscore-dangle: off
+           no-console: off
+           import/no-extraneous-dependencies: off
```

Comment:
**func names:**  looks like we need to use the name for [this](https://github.com/cerner/terra-graphs/blob/main/packages/carbon-graphs/src/js/helpers/label.js#L253) kind of functions which is just a warning so I disabled this linter rule.

**no-underscore-dangle:** we are using underscore dangles in carbon-graphs for some variables to say that "this is a private variable and only used in this method". which I feel is not an issue and also serves some purpose.  

**no-console:** looks like linter does not want us to use console but we are using console.warn in [bubble.js](https://github.com/cerner/terra-graphs/blob/main/packages/carbon-graphs/src/js/controls/Bubble/Bubble.js#L71) just to warn that it is going to deprecated in future release( also in some test cases). 

**import/no-extraneous-dependencies:** Linter wants us to move some of the dependencies into the dependencies list instead of dev dependencies. That is the reason why I turned off those rules.


Please let me know if there is anything that needs to be changed.